reference/client-access/python-sdk.md

```diff
@@ -0,0 +1,287 @@
+---
+description: Technical reference for the libSQL Python client library used to access Turso databases, including sample code.
+keywords:
+  - turso
+  - libsql
+  - python
+  - client
+  - sdk
+  - library
+  - example
+---
+
+# Python SDK
+
+The Python SDK comes with APIs for both synchronous and asynchronous code.
+
+## Installation
+
+Add the [libsql_client module] to your project using `pip`:
+
+```bash
+$ pip install libsql_client
+```
+
+## Initialization
+
+Import the libsql_client module:
+
+```py
+import libsql_client
+```
+
+If you want to use the synchronous API, use the `create_client_sync` function:
+
+```py
+client = libsql_client.create_client_sync(
+    url="libsql://your-database.turso.io"
+    auth_token="your-auth-token"
+)
+```
+
+If you want to use the asynchronous API, use the `create_client` function along
+with `asyncio` and async/await syntax:
+
+```py
+client = libsql_client.create_client_sync(
```

Comment:
This should probably be `create_client()`. It may also be good to point out that this function is sync, people should _not_ call `await libsql_client.create_client()`.

Reply:
Silly mistake, fixed.